pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -616,3 +690,35 @@ def test_rdivmod_invalid(self):
 
         with pytest.raises(TypeError):
             divmod(np.array([22, 24]), td)
+
+    def test_td_div_timedelta_timedeltalike_array(self):
```

Comment:
Would a valid case here be object-dtyped array containing all Timedelta objects (or some mix of timedelta, np.timedelta64)?

Reply:
Done.